packages/strapi-utils/lib/packageManager.js

```diff
@@ -43,8 +43,8 @@ module.exports = {
         }
 
         try {
-          const yarnData = watcher('yarn global ls');
-          isNPM = includes(yarnData, 'strapi');
+          const yarnData = watcher('yarn global list');
+          isNPM = !includes(yarnData, 'strapi');
```

Comment:
`isNPM` should be `false` if is included in the yarn global packages. Can your review it @soupette ?

(I changed `ls` by `list`; the first is deprecated)